lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -183,18 +183,28 @@ def _get_coord_info(self, renderer):
             self.axes.get_ybound(),
             self.axes.get_zbound(),
         ]).T
-        centers = (maxs + mins) / 2.
-        deltas = (maxs - mins) / 12.
-        mins = mins - deltas / 4.
-        maxs = maxs + deltas / 4.
-
-        vals = mins[0], maxs[0], mins[1], maxs[1], mins[2], maxs[2]
-        tc = self.axes.tunit_cube(vals, self.axes.M)
-        avgz = [tc[p1][2] + tc[p2][2] + tc[p3][2] + tc[p4][2]
-                for p1, p2, p3, p4 in self._PLANES]
-        highs = np.array([avgz[2*i] < avgz[2*i+1] for i in range(3)])
-
-        return mins, maxs, centers, deltas, tc, highs
+
+        # Get the mean value for each bound:
+        centers = 0.5 * (maxs + mins)
+
+        # Add a small offset between min/max point and the edge of the
+        # plot:
+        deltas = (maxs - mins) / 12.0
```

Comment:
We're Python 3 now, so no need to `.0`
```suggestion
        deltas = (maxs - mins) / 12
```